### PR TITLE
Testing updated to new approach

### LINKED ISSUE #8: Add continuous integration
(using Cloud Build)

### PR DESCRIPTION
**Pull request by paulpalamarchuk**
* Fix #8

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -12,145 +12,74 @@
...
-	@source test/make.sh && check_bash
-
-.PHONY: check_trailing_whitespace
-check_trailing_whitespace:
-	@source test/make.sh && check_trailing_whitespace
-
-.PHONY: test_check_headers
-test_check_headers:
-	@python test/test_verify_boilerplate.py
+DOCKER_TAG_VERSION_DEVELOPER_TOOLS := 0.4.2
```
Comment (kopachevsky) [inline Makefile]: not related to this PR, but do we have same version in template project?

File: test/setup/iam.tf [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+locals {
+  int_required_roles = [
+    "roles/owner",
+    "roles/pubsub.editor"
```
Comment (kopachevsky) [inline test/setup/iam.tf]: I think owner is overlap pubsub.editor, I guess even replace those 2 with roles/editor would be sufficient

File: test/setup/versions.tf [hunk 3]
```diff
@@ -13,3 +13,15 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
+terraform {
+  required_version = ">= 0.12"
```
Comment (kopachevsky) [inline test/setup/versions.tf]: I thought we agreed to use ~> ?

File: Makefile (hunk 1 above)
Comment (paulpalamarchuk) [reply Makefile]: No, but it's the latest version of the image at the moment.

File: test/setup/iam.tf (hunk 2 above)
Comment (paulpalamarchuk) [reply test/setup/iam.tf]: Removed "owner" role

File: test/setup/versions.tf (hunk 3 above)
Comment (paulpalamarchuk) [reply test/setup/versions.tf]: updated

**Pull request merged**